crates/ide-completion/src/render/function.rs

```diff
@@ -324,7 +324,12 @@ fn detail(ctx: &CompletionContext<'_>, func: hir::Function) -> String {
         format_to!(detail, "unsafe ");
     }
 
-    format_to!(detail, "fn({})", params_display(ctx, func));
+    format_to!(
+        detail,
+        "fn({}{})",
+        params_display(ctx, func),
+        if func.is_varargs(ctx.db) { ", ..." } else { "" }
```

Comment:
Let's add that change to `params_display`. Might even be better to change this to not have `params_display` allocate a separate string here in the first place, that is use `format_to` for `params_display` (instead of `format!`).

Reply:
Done. Passing a mutable string to `params_display` seemed a bit messy to me compared to the gains, so I left as-is.